extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/HueBridge.java

```diff
@@ -525,6 +564,26 @@ public void setGroupLights(Group group, List<HueObject> lights) throws IOExcepti
         handleErrors(result);
     }
 
+    /**
+     * Get details for specified group.
+     *
+     * @param id id of the group
+     * @return FullGroup the group
+     * @throws IOException
+     * @throws ApiException
+     */
+    public FullGroup getGroupAttributes(String id) throws IOException, ApiException {
```

Comment:
This method is essentially identical to `#getGroup`, it might be worthwhile to extract from its body.

Reply:
Thanks, i resused the body and thus the enc-function is called, too.